src/main/java/mate/academy/resources/log4j2.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
does it work? can you share the screenshots of the file and the console with logs?